Mono.CairoWarp/WarpBilinear.cs

```diff
@@ -20,9 +19,6 @@ public WarpBilinear(IEnumerable<PointD> destPoints, Rectangle srcRect)
 			if (destPoints == null || destPoints.Count() != 4)
 				throw new ArgumentException("destPoints");
 
-			if (srcRect == null)
-				throw new ArgumentNullException("srcRect");
-
```

Comment:
why?

Reply:
Rectangle is a struct, never will be null

Reply:
ack